app/classifier/tasks/flexible-survey/choice.cjsx

```diff
@@ -0,0 +1,153 @@
+React = require 'react'
+TriggeredModalForm = require 'modal-form/triggered'
+{Markdown} = require 'markdownz'
+Utility = require './utility'
+
+ImageFlipper = React.createClass
+  displayName: 'ImageFlipper'
+
+  getDefaultProps: ->
+    images: []
+
+  getInitialState: ->
+    frame: 0
+
+  PRELOAD_STYLE:
+    height: 0
+    overflow: 'hidden'
+    position: 'fixed'
+    right: 0
+    width: 0
+
+  render: ->
+    <span className="survey-task-image-flipper">
+      {@renderPreload()}
+      <img src={@props.images[@state.frame]} className="survey-task-image-flipper-image" />
+      <span className="survey-task-image-flipper-pips">
+        {unless @props.images.length is 1
+          for index in [0...@props.images.length]
+            <span key={@props.images[index]}>
+              <button type="button" className="survey-task-image-flipper-pip" disabled={index is @state.frame} onClick={@handleFrameChange.bind this, index}>{index + 1}</button>
+              {' '}
+            </span>}
+      </span>
+    </span>
+
+  renderPreload: ->
+    <div style={@PRELOAD_STYLE}>
```

Comment:
This style is required for the module to function right, so I think including it in the module is the right call.